src/coverlet.core/Logging/ILogger.cs

```diff
@@ -1,11 +1,15 @@
-namespace Coverlet.Console.Logging
+namespace Coverlet.Core
```

Comment:
Minor namespace nagging: should be `Coverlet.Core.Logging` to be consistent with directory structure.

Reply:
Perfectly acceptable.  Fixed